data/apply-planet_osm_polygon.sql

```diff
@@ -13,30 +13,26 @@ CREATE INDEX planet_osm_polygon_admin_level_geom_index ON planet_osm_polygon USI
-    WHERE coalesce("aerialway", "aeroway", "amenity", "barrier", "craft", "emergency", "highway", "historic", "leisure", "lock", "man_made", "natural", "office", "power", "railway", "shop", "tourism", "waterway", "tags"->'rental', "tags"->'zoo', "tags"->'attraction', tags->'healthcare') IS NOT NULL;
+    mz_poi_min_zoom = mz_calculate_min_zoom_pois(planet_osm_polygon.*)
+    WHERE mz_calculate_min_zoom_pois(planet_osm_polygon.*) IS NOT NULL;
 
-CREATE INDEX planet_osm_polygon_is_landuse_col_index ON planet_osm_polygon(mz_is_landuse) WHERE mz_is_landuse=TRUE;
+CREATE INDEX planet_osm_polygon_landuse_geom_9_index ON planet_osm_polygon USING gist(way) WHERE mz_landuse_min_zoom <= 9;
+CREATE INDEX planet_osm_polygon_landuse_geom_12_index ON planet_osm_polygon USING gist(way) WHERE mz_landuse_min_zoom <= 12;
+CREATE INDEX planet_osm_polygon_landuse_geom_15_index ON planet_osm_polygon USING gist(way) WHERE mz_landuse_min_zoom <= 15;
 
-CREATE INDEX planet_osm_polygon_landuse_geom_9_index ON planet_osm_polygon USING gist(way) WHERE mz_is_landuse=TRUE AND mz_landuse_min_zoom <= 9;
-CREATE INDEX planet_osm_polygon_landuse_geom_12_index ON planet_osm_polygon USING gist(way) WHERE mz_is_landuse=TRUE AND mz_landuse_min_zoom <= 12;
-CREATE INDEX planet_osm_polygon_landuse_geom_15_index ON planet_osm_polygon USING gist(way) WHERE mz_is_landuse=TRUE AND mz_landuse_min_zoom <= 15;
-
-CREATE INDEX planet_osm_polygon_landuse_boundary_geom_4_index ON planet_osm_polygon USING gist(way) WHERE mz_is_landuse=TRUE AND (boundary IN ('national_park', 'protected_area') OR leisure='nature_reserve') AND mz_landuse_min_zoom <= 4;
-CREATE INDEX planet_osm_polygon_landuse_boundary_geom_6_index ON planet_osm_polygon USING gist(way) WHERE mz_is_landuse=TRUE AND (boundary IN ('national_park', 'protected_area') OR leisure='nature_reserve') AND mz_landuse_min_zoom <= 6;
-CREATE INDEX planet_osm_polygon_landuse_boundary_geom_8_index ON planet_osm_polygon USING gist(way) WHERE mz_is_landuse=TRUE AND (boundary IN ('national_park', 'protected_area') OR leisure='nature_reserve') AND mz_landuse_min_zoom <= 8;
+CREATE INDEX planet_osm_polygon_landuse_boundary_geom_4_index ON planet_osm_polygon USING gist(way) WHERE mz_landuse_min_zoom IS NOT NULL AND (boundary IN ('national_park', 'protected_area') OR leisure='nature_reserve') AND mz_landuse_min_zoom <= 4;
+CREATE INDEX planet_osm_polygon_landuse_boundary_geom_6_index ON planet_osm_polygon USING gist(way) WHERE mz_landuse_min_zoom IS NOT NULL AND (boundary IN ('national_park', 'protected_area') OR leisure='nature_reserve') AND mz_landuse_min_zoom <= 6;
+CREATE INDEX planet_osm_polygon_landuse_boundary_geom_8_index ON planet_osm_polygon USING gist(way) WHERE mz_landuse_min_zoom IS NOT NULL AND (boundary IN ('national_park', 'protected_area') OR leisure='nature_reserve') AND mz_landuse_min_zoom <= 8;
```

Comment:
Do we need both `mz_landuse_min_zoom IS NOT NULL` and `mz_landuse_min_zoom <= N` in the lines above? As I understand it, `NULL <= N` will also be `NULL`, which is falsey in the where clause.